package.json

```diff
@@ -2,17 +2,23 @@
   "name": "awesome-imba",
   "version": "1.0.0",
```

Comment:
not sure about current version for this repo... I didn't track all changes before.

Reply:
I guess we can check all PRs and decide about version